From ea6865e1bfc1a1ac72fe95be42abb88b47413363 Mon Sep 17 00:00:00 2001
From: Oskari Lemmela <oskari@lemmela.net>
Date: Tue, 23 Oct 2018 21:53:29 +0300
Subject: [PATCH 111/146] mfd: axp20x: Add AC power supply cell for AXP813

As axp20x-ac-power-supply now supports AXP813, add a cell for it.

Signed-off-by: Oskari Lemmela <oskari@lemmela.net>
Reviewed-by: Quentin Schulz <quentin.schulz@bootlin.com>
Reviewed-by: Chen-Yu Tsai <wens@csie.org>
Tested-by: Vasily Khoruzhick <anarsoul@gmail.com>

Changed-by: Igor@armbian
---
 drivers/mfd/axp20x.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/mfd/axp20x.c b/drivers/mfd/axp20x.c
index 0be511dd93d0..dfc3cff1d08b 100644
--- a/drivers/mfd/axp20x.c
+++ b/drivers/mfd/axp20x.c
@@ -805,8 +805,13 @@ static const struct mfd_cell axp813_cells[] = {
 		.name			= "axp813-adc",
 		.of_compatible		= "x-powers,axp813-adc",
 	}, {
-		.name		= "axp20x-battery-power-supply",
-		.of_compatible	= "x-powers,axp813-battery-power-supply",
+		.name			= "axp20x-battery-power-supply",
+		.of_compatible		= "x-powers,axp813-battery-power-supply",
+	}, {
+		.name			= "axp20x-ac-power-supply",
+		.of_compatible		= "x-powers,axp813-ac-power-supply",
+		.num_resources		= ARRAY_SIZE(axp20x_ac_power_supply_resources),
+		.resources		= axp20x_ac_power_supply_resources,
 	}, {
 		.name			= "axp20x-usb-power-supply",
 		.of_compatible		= "x-powers,axp813-usb-power-supply",

-- 
2.17.1

